src/travel-details-screens/utils.ts

```diff
@@ -506,7 +505,7 @@ export function getLineAndTimeA11yLabel(
       : undefined,
   ]
     .filter(isDefined)
-    .join(screenReaderPause);
```

Comment:
Shouldn't think this would cause a test to fail, but due to imports dependent on theme in `@atb/components/text`, and the "ChangeNativeBridge module is not linked" check, it did 🫠